tox.ini

```
@@ -19,3 +19,12 @@ deps =
     yamllint==1.15.0
 commands =
     bash -c "yamllint eve/main.yml salt/**/*.yaml"
+
+[testenv:tests]
+deps =
+    -r {toxinidir}/tests/requirements.txt
```

Testing this PR. On my `tox` and `pip`, this can't have a space, needs to be
```suggestion
    -r{toxinidir}/tests/requirements.txt
```